Example/CryptoOnramp Example/CryptoOnramp Example/RegistrationView.swift

```diff
@@ -149,6 +162,57 @@ struct RegistrationView: View {
             }
         }
     }
+
+    private func presentVerification(using coordinator: CryptoOnrampCoordinator) async -> String? {
+        if let viewController = UIApplication.shared.findTopNavigationController() {
+            do {
+                let result = try await coordinator.authenticateUser(from: viewController)
+                isLoading.wrappedValue = false
```

Comment:
Since`isLoading.wrappedValue = false` is modifying view state, I think it should be run on the main actor.